python/lib/dicom/dicom_log.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
+import os
+import socket
+from lib.dicom.text_dict import DictWriter
+
+
+class Log:
```

Comment:
there is already a class `Log` that exists. I would rename this one `DicomLog` instead to make it clear this is for the DICOM log file.

Reply:
That makes sense, renamed to `DicomArchiveLog` !